plugins/woocommerce-blocks/assets/js/base/components/country-input/billing-country-input.tsx

```diff
@@ -10,8 +10,7 @@ import CountryInput from './country-input';
 import type { CountryInputProps } from './CountryInputProps';
 
 const BillingCountryInput = ( props: CountryInputProps ): JSX.Element => {
-	//  TODO - are errorMessage and errorId still relevant when select always has a value?
-	const { errorMessage: _, errorId: __, ...restOfProps } = props;
+	const { ...restOfProps } = props;
```

Comment:
We now pass the errorMessage and id through to perform validation in the CountryInput